MCPB as the preferred method for installing MCP in Claude Desktop (#8445)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-ai/integrate-mcp-claude.md

1. Add the MCP server:

  ```bash
  claude mcp add dbt -- uvx dbt-mcp
  ```
2. Open the configuration editor:

  ```bash
  claude mcp edit dbt
  ```

3. In the configuration editor, add your environment variables based on your use case:

<Tabs>
<TabItem value="CLI only">

For <Constant name="core" /> or <Constant name="fusion" /> only (no <Constant name="dbt_platform" />):
```json
{
  "command": "uvx",
  "args": ["dbt-mcp"],
  "env": {
    "DBT_PROJECT_DIR": "/path/to/your/dbt/project",
    "DBT_PATH": "/path/to/your/dbt/executable"
  }
}
```

</TabItem>
<TabItem value="OAuth with dbt platform">

For OAuth authentication (requires static subdomain):
```json
{
  "command": "uvx",
  "args": ["dbt-mcp"],
  "env": {
    "DBT_HOST": "https://your-subdomain.us1.dbt.com",
    "DBT_PROJECT_DIR": "/path/to/your/dbt/project",
    "DBT_PATH": "/path/to/your/dbt/executable"
  }
}
```

</TabItem>
</Tabs>

#### Using an `.env` file

If you prefer to manage environment variables in a separate file:

```bash
claude mcp add dbt -- uvx --env-file <path-to-.env-file> dbt-mcp
```
Replace `<path-to-.env-file>` with the full path to your `.env` file.

## Claude Desktop

<StaticSubdomainRequired />

> 

Reviewer: ```suggestion

To configure Claude Desktop to use the dbt MCP server:
```

1. Add the MCP server:

  ```bash
  claude mcp add dbt -- uvx dbt-mcp
  ```
2. Open the configuration editor:

  ```bash
  claude mcp edit dbt
  ```

3. In the configuration editor, add your environment variables based on your use case:

<Tabs>
<TabItem value="CLI only">

For <Constant name="core" /> or <Constant name="fusion" /> only (no <Constant name="dbt_platform" />):
```json
{
  "command": "uvx",
  "args": ["dbt-mcp"],
  "env": {
    "DBT_PROJECT_DIR": "/path/to/your/dbt/project",
    "DBT_PATH": "/path/to/your/dbt/executable"
  }
}
```

</TabItem>
<TabItem value="OAuth with dbt platform">

For OAuth authentication (requires static subdomain):
```json
{
  "command": "uvx",
  "args": ["dbt-mcp"],
  "env": {
    "DBT_HOST": "https://your-subdomain.us1.dbt.com",
    "DBT_PROJECT_DIR": "/path/to/your/dbt/project",
    "DBT_PATH": "/path/to/your/dbt/executable"
  }
}
```

</TabItem>
</Tabs>

#### Using an `.env` file

If you prefer to manage environment variables in a separate file:

```bash
claude mcp add dbt -- uvx --env-file <path-to-.env-file> dbt-mcp
```

> Replace `<path-to-.env-file>` with the full path to your `.env` file.

Reviewer 2: 🔥

## Claude Desktop

<StaticSubdomainRequired />

> 1. Go to the latest dbt MCP release [here](https://github.com/dbt-labs/dbt-mcp/releases/latest

Reviewer: ```suggestion
1. Go to the [latest dbt MCP release](https://github.com/dbt-labs/dbt-mcp/releases/latest) and download the `dbt-mcp.mcpb` file.
```

> ) and download the `dbt-mcp.mcpb` file.

Reviewer: ```suggestion
```

> 2. After this is downloaded, double-click on this file to open it in Claude Desktop.

Reviewer: ```suggestion
2.  Double-click the downloaded file to open it in Claude Desktop.
```

> 3. Follow the directions to set the `dbt Platform Host`. You can find this in dbt platform by navigating to Dashboard, then Settings.

Reviewer: ```suggestion
3. Configure the **dbt Platform Host**. You can find this in your <Constant name="dbt_platform" /> account by navigating to **Dashboard** > **Settings* and scrolling to the **Semantic Layer** section.
```

4. Enable the server in Claude Desktop.

> 5. Ask Claude a data-related question and observe dbt MCP being utilized.

Reviewer: ```suggestion
5. Ask Claude a data-related question and see dbt MCP in action!
```

### Advanced configuration with Claude Desktop

> 

Reviewer: ```suggestion

To add advanced configurations:
```
